zinc/src/sbt-test/source-dependencies/trait-trait-211/mirtest/Hello.scala

```diff
@@ -0,0 +1,14 @@
+package xx
+
+import gg.table._
+
+object Hello extends App {
+  def testEval: Unit = {
+    val consumer = new StringLibSpecs {}
```

Comment:
do you need this local inheritance relationship to trigger the bug?

Reply:
no. I can repro with this:

```scala
package xx

object Hello extends App {
  val consumer = new EvaluatorSpecification {}
}
```